### ISSUE TITLE
Rename repos to NLog.ActiveMQ

### ISSUE THREAD

**Issue by 304NotModified** [NLog/NLog.Contrib.ActiveMQ#1]

**Comment by v-karbovnichy** [comment]
Maybe rename to NLog.Targets.ActiveMQ instead?

**Comment by 304NotModified** [comment]
yeah also possible, although we have a NLog.Web, NLog.Windows.Forms and NLog.ETW etc.

**Comment by snakefoot** [comment]
Seems it has been raced by someone else:

- https://www.nuget.org/packages/NLog.Targets.ActiveMQ

And actively maintained, and in better shape than this package (Handles NetStandard and better resource-usage and better LICENSE). Maybe just obsolete this and redirect in nuget?
